pkg/util/testlogger/testlogger.go

```diff
@@ -33,9 +37,12 @@ func (tl *TestLogger) Log(level logging.LogLevel, text string, args ...interface
 		text:  text,
 		args:  args,
 	}
+	tl.internalMutex.Lock()
+	defer tl.internalMutex.Unlock()
 	// TODO: With this implementation, duplicate entries are overwritten. Make duplicates possible.
```

Comment:
Do we now support duplicate entries? If yes, this comment can be removed.

Reply:
I think we still do not in the testlogger.